packages/design-system/src/atoms/icon/index.tsx

```diff
@@ -46,6 +46,7 @@ import Layer from '../../assets/icons/layer.svg'
 import Link from '../../assets/icons/link.svg'
 import Link2 from '../../assets/icons/link-2.svg'
 import Lock from '../../assets/icons/lock.svg'
+import RollLogo from '../../assets/icons/roll-iso.svg'
```

Comment:
Could you update the filename as well, please? Just to be consistent :)

```suggestion
import RollLogo from '../../assets/icons/roll-logo.svg'
```